### PR TITLE
python tests: improves test_distance.py

### PR DESCRIPTION
**Pull request by PetholzA**
this is #1077 renewed, because of problems. some of the comments remain to be adressed

### CODE REVIEW

**Comment by fabratu** [comment]
Coverage is currently at 70%. Please try to increase to at least 80%, better 90%.

File: networkit/test/test_distance.py [hunk 1]
```diff
@@ -188,11 +229,49 @@ def testMultiTargetSTSP(self):
...
+				if g.isWeighted():
+					algoDyn = nk.distance.DynDijkstra(g, source)
+				else:
+					algoDyn = nk.distance.DynBFS(g, source)
+				algoDyn.run()
+				if not g.isWeighted():
+					batch1 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.EDGE_ADDITION, 3, 7, 1.0)
+					batch2 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.EDGE_ADDITION, 1, 5, 1.0)
+					batch = [batch1,batch2]
+					algoDyn.updateBatch(batch)
```
Comment (fabratu) [inline networkit/test/test_distance.py:272]: Here, you check only for `updateBatch` - is `update` already tested?

File: networkit/test/test_distance.py [hunk 2]
```diff
@@ -233,5 +312,44 @@ def testDynPrunedLandmarkLabeling(self):
...
+	
+	def testVolume(self):
+		vol = nk.distance.Volume()
+		#single instance
+		self.assertEqual(vol.volume(self.L, 0.5), 1.0)
+		#list test
+		self.assertListEqual(vol.volume(self.L, [0.1, 0.2, 0.8]), [1.0, 1.0, 1.0])
+		self.assertRaises(Exception, vol.volume(self.L, "wrongType"))
+
+	def testWithNonConsecutiveNodeLabelsAPSP(self):
```
Comment (fabratu) [inline networkit/test/test_distance.py:347]: Why is this test done?

**Review by fabratu — CHANGES_REQUESTED** [review]
While you are at it, can you also add a test similar to `testAsArrayAPSP` for SPSP?

File: networkit/test/test_distance.py (hunk 1 above)
Comment (PetholzA) [reply networkit/test/test_distance.py:306]: update() had been used in testDynPrunedLandmarkLabeling, therefore, i decided to only test updateBatch()

File: networkit/test/test_distance.py (hunk 2 above)
Comment (PetholzA) [reply networkit/test/test_distance.py]: it was a leftover, i removed it

**Review by fabratu — APPROVED** [review]

**Pull request merged**